### ISSUE TITLE
The table of content should be an output, not an input

### ISSUE THREAD

**Issue by Villetaneuse** [Zimmi48/platform-docs#10]
We should only give (unnumbered) titles, subtitles, etc and, if need be, the numbering and the table of content should be automatically written by some tool.

I can write some crude `sh`/`awk`/`sed` helpers, but maybe the format should stabilize before.

**Comment by thomas-lamiaux** [comment]
I agree. We are most likely to keep with coqdoc for a while, so it might be worth it doing something now.
@Zimmi48 ?
